Sources/Sentry/SentryScope.m

```diff
@@ -558,6 +563,12 @@ - (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
             newContext[@"trace"] = [span serialize];
         }
     }
+
+    if (event.exceptions.count > 0 && newContext[@"trace"] == nil) {
```

Comment:
Is there are reason this is being done depending on the exception count? `sentry-java` only checks [the trace context](https://github.com/getsentry/sentry-java/blob/fb296f0fa15044af53a7c6d62d168b732eb16500/sentry/src/main/java/io/sentry/SentryClient.java#L677-L685).

Reply:
The count check is only to be sure this is an error event. If I got it right we dont send trace for capture message. But I just confirm that capture would not run this code anyway.